### DISCUSSION TITLE
Smartnaming

### DISCUSSION THREAD

**Discussion by ontamomo** [aclap-dev/video-downloadhelper#1050] [Help and Q&A]
### Your question

Great software! I was wondering how the smartnaming works. "text from CSS selector (optional)", does this mean that you can set the video-downloadhelper to select part of the webpages content and put it in the title of the downloaded file. And it will do so consistently if you use the same page. How would I go about finding this CSS selector?

Apologize for the tech illiteracy

### Addon & CoApp version and details

Great software! I was wondering how the smartnaming works. "text from CSS selector (optional)", does this mean that you can set the video-downloadhelper to select part of the webpages content and put it in the title of the downloaded file. And it will do so consistently if you use the same page. How would I go about finding this CSS selector?

Apologize for the tech illiteracy

**Comment by GrampaWildWilly** [comment]
You'll want to study discussion #838.  That will show you how it's supposed to work.  Sadly, it's completely broken at the moment.  But advanced study will prepare you for the day when it is fixed.

**Reply by ontamomo** [reply]
Thanks a lot, a lot of useful information there. Maybe I should revert to the older version?

**Reply by paulrouget** [reply]
It should work now in the beta version if you use Firefox.
